enterprise/server/suggestion/suggestion.go

```diff
@@ -0,0 +1,151 @@
+package suggestion
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/eventlog"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	elpb "github.com/buildbuddy-io/buildbuddy/proto/eventlog"
+	supb "github.com/buildbuddy-io/buildbuddy/proto/suggestion"
+)
+
+var apiKey = flag.String("openai.api_key", "", "OpenAI API key")
+var model = flag.String("openai.model", "gpt-3.5-turbo", "OpenAI model name to use, if not set 'gpt-3.5-turbo' is used.")
+
+const (
+	minLines = 1000
+	maxChars = 8000
```

Comment:
nit: could use a comment here about what these are and how to set them

Reply:
Addressed in: https://github.com/buildbuddy-io/buildbuddy/pull/3583/commits/2f05336a0f6f4d071e240592e93e459888a62324